plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineProjectDeployer.java

```diff
@@ -25,49 +20,19 @@
  */
 public class AppEngineProjectDeployer {
 
-  private static final String PLUGIN_ID = "com.google.cloud.tools.eclipse.appengine.localserver"; //$NON-NLS-1$
-  private static final String APPENGINE_WEB_XML_PATH = "WEB-INF/appengine-web.xml"; //$NON-NLS-1$
-
-  private AppEngineDeployInfo deployInfo;
-
-  public AppEngineProjectDeployer() {
-    deployInfo = new AppEngineDeployInfo();
-  }
-
-  @VisibleForTesting
-  AppEngineProjectDeployer(AppEngineDeployInfo deployInfo) {
-    this.deployInfo = deployInfo;
-  }
-
-  public void deploy(IPath stagingDirectory, CloudSdk cloudSdk, IProgressMonitor monitor) throws CoreException {
+  public void deploy(IPath stagingDirectory, CloudSdk cloudSdk,
```

Comment:
Currently it is in the workspace, see `com.google.cloud.tools.eclipse.appengine.deploy.ui.standard.StandardDeployCommandHandler.getTempDir()`.